packages/apps/plugins/plugin-settings/src/components/SettingsDialog.tsx

```diff
@@ -35,7 +35,7 @@ export const SettingsDialog = ({
     .sort(({ name: a }, { name: b }) => a?.localeCompare(b ?? '') ?? 0);
 
   return (
-    <Dialog.Content classNames={['h-full md:max-is-[40rem] overflow-hidden']}>
+    <Dialog.Content classNames={['h-auto h-max-full md:max-is-[40rem] overflow-hidden']}>
```

Comment:
```suggestion
    <Dialog.Content classNames={['bs-content max-bs-full md:max-is-[40rem] overflow-hidden']}>
```
Whether this will limit its size depends on the layout (I’m forgetting), a `max-bs` rule might be unnecessary or have no effect.